src/inner_product_proof.rs

```diff
@@ -338,6 +338,7 @@ impl InnerProductProof {
     /// The layout of the inner product proof is:
     /// * \\(n\\) pairs of compressed Ristretto points \\(L_0, R_0 \dots, L_{n-1}, R_{n-1}\\),
     /// * two scalars \\(a, b\\).
+    #[allow(dead_code)]
```

Comment:
Hmm, just to clarify, this is suppressing an existing warning, not a new warning that appeared as a result of these changes?

Reply:
Correct